### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by PatchaSravani2** [bluenviron/mediamtx#2830] [Questions]
### Question

I'm streaming RTMP stream and viewing it using hls in my chrome browser and also I'm using ngrok.
I configured tunnels for 1935 and 8888 in ngrok with tcp protocol. The issue is that if I use http it is working fine but if I use https in source I'm getting this error in my chrome console.
Error: Failed to load resource: net::ERR_SSL_PROTOCOL_ERROR.

**Comment by deep-wb** [comment]
The "ERR_SSL_PROTOCOL_ERROR" suggests an issue with SSL when using HTTPS for RTMP streaming via Ngrok. Verify Ngrok's HTTPS configuration, check SSL certificates, and ensure your RTMP source uses "https://". [Read more](http://bit.ly/FixSSLCertificateError).

[a comment by an automated account is not shown]
